langchain/llms/openai.py

```diff
@@ -242,10 +242,15 @@ def _default_params(self) -> Dict[str, Any]:
             "frequency_penalty": self.frequency_penalty,
             "presence_penalty": self.presence_penalty,
             "n": self.n,
-            "best_of": self.best_of,
             "request_timeout": self.request_timeout,
             "logit_bias": self.logit_bias,
         }
+
+        # Azure gpt-35-turbo doesn't support best_of
```

Comment:
is this meant to be "Azure gpt-35-turbo doesn't support best_of equal to 1"

Reply:
My interpretation was that "best_of = 1" is the default and azure turbo doesn't support best_of customization at all. I'd prefer a check at validation time

Reply:
in that case should we explicitly check if model name is gpt-35-turbo and best_of>1 and raise error if so? similar to check in line 231

Reply:
that's ok as if you set best_of > 1, you will a nice error from Azure says it is not supported. My change is intentional not handling that case.